sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -303,6 +304,51 @@
   color: #16c92e;
 }
 
+/* form.io status chip */
+.formio-status-chip {
+  border-width: 1px;
+  padding-left: 15px;
+  padding-right: 10px;
+  position: absolute;
+  border-radius: 30px;
+  border-style: solid;
+  text-transform: uppercase;
+  font-weight: 700 !important;
```

Comment:
There is a variable called $formio-header-wt  in the global-style-variable.scss file which can be used here.

Reply:
Renamed the variable $formio-header-wt to $strong-font-weight and consumed everywhere.